src/nvim/fileio.c

```diff
@@ -5992,18 +6011,20 @@ void ex_doautoall(exarg_T *eap)
   check_cursor();           /* just in case lines got deleted */
 }
 
-/*
- * Check *argp for <nomodeline>.  When it is present return FALSE, otherwise
- * return TRUE and advance *argp to after it.
- * Thus return TRUE when do_modelines() should be called.
- */
-int check_nomodeline(char_u **argp)
+/// Check *argp for <nomodeline>.  When it is present return false, otherwise
+/// return true and advance *argp to after it.
+///
+/// @param[in,out] argp argument string
+///
+/// @return true when do_modelines() should be called.
```

Comment:
I just mean this change splits the information about the return value in two parts, for no reason. Becuse the return value is related to the side-effect (argp), the _simplest_ fix would be to just move back this to the describtion. (but if you prefere you can edit it otherwise, as long as the split is resolved)